contracts/src/chainlink/__init__.py

```diff
@@ -0,0 +1,11 @@
+"""StarkNet/Cairo development toolbelt."""
```

Comment:
Why is this file under this specific path?

Could we contain all/most of these new files under a subfolder like `contracts/tools/python`?

Reply:
Looks like that isn't supplied out of the box
https://stackoverflow.com/questions/25425511/python-distutils-setup-relative-paths-for-package-dir